tests/integration/guided_with_novel.json

```
@@ -0,0 +1,20 @@
+{
+  "MultiBamExpressionQuantification.strandedness": "None",
+  "MultiBamExpressionQuantification.referenceGtfFile": "tests/data/reference.gtf",
```

This is missing one input:
```suggestion
  "MultiBamExpressionQuantification.referenceGtfFile": "tests/data/reference.gtf",
  "MultiBamExpressionQuantification.detectNovelTranscripts": true,
```

---

Whoops must have been a bit distracted.